### PR TITLE
ItemStacks rendering on the background of slots. Individually Colorable Slots

### PR DESCRIPTION
**Pull request by Lanse505**

### CODE REVIEW

**Comment by Lanse505** [comment]
This PR should be done now @Buuz135 @SkySom
It adresses and fixes #4 #5

**Comment by Lanse505** [comment]
Just confirmed, renders Chests just fine, same for enchanted tools, banners, etc.

**Comment by Lanse505** [comment]
bild

bild

**Review by SkySom — APPROVED** [review]

File: src/main/java/com/hrznstudio/titanium/component/inventory/InventoryComponent.java [hunk 1]
```diff
@@ -219,6 +240,71 @@ public T getComponentHarness() {
...
+    }
+
+    /**
+     * @param slot
+     * @param color
+     * @return
+     */
+    public InventoryComponent<T> setSlotToColorRender(int slot, int color) {
+        if (!this.colorGuiEnabled) {
+            this.colorGuiEnabled = true;
```
Comment (Buuz135) [inline src/main/java/com/hrznstudio/titanium/component/inventory/InventoryComponent.java]: If SidedInvComp uses the Setter, this should too, same with others so everything uses the same

**Review by Buuz135 — APPROVED** [review]

**Pull request merged**